api_v3/services/LiveStreamService.php

```diff
@@ -196,9 +196,13 @@ private function validateMaxStreamsNotReached(LiveEntry $liveEntry)
 		$maxPassthroughStreams = $liveEntryPartner->getMaxLiveStreamInputs();
 		KalturaLog::debug("Max Passthrough streams [$maxPassthroughStreams]");
 		$adminTagsOrigLimits = $liveEntryPartner->getMaxConcurrentLiveByAdminTag();
-		KalturaLog::debug('Current AdminTags: [' . $liveEntry->getAdminTags() . '] AdminTag limits : [' . print_r($adminTagsOrigLimits, true) . ']');
-		$adminTagsCounters = $adminTagsOrigLimits;
+		$isCloudTranscode = $this->isCloudTranscode($liveEntry->getConversionProfileId());
 		
+		$isAdminTagLimited = $this->isAdminTagLimited($liveEntry, array_keys($adminTagsOrigLimits));
```

Comment:
Can move that to the "if" as this bool is not use afterward.
And prefer not to have var and function with the same name as here